test/gpu/native/kernelFnCalls/nilInFn.chpl

```diff
@@ -8,7 +8,7 @@ proc foo(ref x) {
 on here.gpus[0] {
   var x: c_ptr(int);
   @assertOnGpu
-  foreach i in 1..10 {
+  foreach i in 1..10 with (in x) {
```

Comment:
Could you explain this one?

Reply:
`foo` takes in `x` as a `ref` so I need to give `x` a non-const intent. I think I could just as well make it `ref` rather than `in` (for the purpose of the test I don't think it matters).

The test is a bit strange in that `x` starts nil, gets processed in a `foreach` loop that will overwrite it with `nil` and then we `writeln` it so whether `x` comes in as `ref` or `in` is inconsequential.